aries-site/src/data/wcag/tag.json

```diff
@@ -0,0 +1,512 @@
+    "notes": ""
+  },
+  {
+    "rule": "1.2.8",
+    "label": "Media Alternative (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.9",
+    "label": "Audio-only (Live)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.1",
+    "label": "Info and Relationships",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.2",
+    "label": "Meaningful Sequence",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.3",
+    "label": "Sensory Characteristics",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.4",
+    "label": "Orientation",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.5",
+    "label": "Identify Input Purpose",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.6",
+    "label": "Identify Purpose",
+    "status": "failed",
+    "notes": "The close icon doesn't have a descriptive aria-label"
+  },
+  {
+    "rule": "1.4.1",
+    "label": "Use of Color",
+    "status": "conditional",
+    "notes": "In the highlight tag variant we use color to communicate meaning. Ideally this information should also be available via text."
+  },
+  {
+    "rule": "1.4.2",
+    "label": "Audio Control",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.3",
+    "label": "Contrast (Minimum)",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.4",
+    "label": "Resize Text",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.5",
+    "label": "Images of Text",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.6",
+    "label": "Contrast (Enhanced)",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.7",
+    "label": "Low or No Background Audio",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.8",
+    "label": "Visual Presentation",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.9",
+    "label": "Images of Text (No Exception)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.10",
+    "label": "Reflow",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.11",
+    "label": "Non-text Contrast",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.12",
+    "label": "Text Spacing",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.13",
+    "label": "Content on Hover or Focus",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.1.1",
+    "label": "Keyboard",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.1.2",
+    "label": "No Keyboard Trap",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.1.3",
+    "label": "Keyboard (No Exception)",
+    "status": "passed",
+    "notes": ""
+  },
```

Comment:
what about rule 2.1.4 character key shortcuts

Reply:
Good catch, added this in latest commit